### PR TITLE
Docker/Readme: codestyle cleanup (9af6bf1 follow-up)

### PR DESCRIPTION
**Pull request by ghost**
**Description:**

This is a follow-up to PR #25832 / commit 9af6bf15aa2fe836c3ebba306eaaa8971f00fac4 to offer some documentation copy review.

**Changes proposed:**

- Simplify MD back tick usage in stand-alone horizontal strings.
- Sentence grammar improvements/adjustments (might->may etc.)
- Punctuation: Add a period dot at the end of proper sentences.
- Punctuation: Add Oxford commas for natural readability.
- Punctuation: Add colon at the end of sentence introducing command examples.
- Reduce indentation level for bullet point annotation from second level to first level.
- Spell Linux with a capital L (proper noun, from "Linus + UNIX").
- UNIX is a proper noun with all its letters in uppercase.
- Add missing double quote character at the end of the .conf file reference.
- ~~Remove a stray double quote character from the start of a command.~~
- ~~Replace HTML entity `&#46;` with its ASCII representation, a period dot.~~

Whitespace changes:
- Add editorial blank line below H1 (page title) and H2 headings.
- Add editorial blank line below MD code block fences (codestyle).

**Target branch(es):** 3.3.5/master

- [X] 3.3.5
- [ ] master

**Issues addressed:**

None that I know of.

**Tests performed:**

This is just a README edit, and does not interfere with any binary build whatsoever.

**Known issues and TODO list:** (add/remove lines as needed)

- [x] Some of the formatting changes are open to discussion, in case of any layout changes.
- [x] Some of the phrasing may be open to discussion, in case of changes in meaning.

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
- [x] **Regular source view:** https://github.com/TrinityCore/TrinityCore/pull/25864/files
- [x] **Hide whitespace changes:** https://github.com/TrinityCore/TrinityCore/pull/25864/files?diff=unified&w=1 (reduces the review length ever so slightly)
- [x] **Rich Diff view:** https://github.com/TrinityCore/TrinityCore/pull/25864/files?diff=unified&w=1&short_path=f042faa

**Comment by illfated** [comment]
#sign-off

File: contrib/Docker/README.md [hunk 1]
```diff
@@ -1,54 +1,62 @@
...
-        docker run --entrypoint=authserver -it --volume=/host/path/to/configs:/home/circleci/project/bin/check_install/etc --volume=/var/run/mysqld/mysqld.sock:/var/run/mysqld/mysqld.sock -p=3724:3724 "image name"
-        ```
-    - worldserver
-        ```
-        docker run --entrypoint=worldserver -it --volume=/host/path/to/configs:/home/circleci/project/bin/check_install/etc --volume=/var/run/mysqld/mysqld.sock:/var/run/mysqld/mysqld.sock --volume=/path/to/data/directory:/trinity/data -p=8085:8085 "image name"
-        ```
-    Change the ports and other parameters as needed. Consult Docker documentation for additional details about possible configuration settings.
+
+1. Copy the .conf files from the TrinityCore GitHub repository to a local folder which will be passed on as a mapped volume to docker.
+1. Set the MySQL host in the .conf files to use the UNIX socket of MySQL, i.e.: `.;/var/run/mysqld/mysqld.sock;username;password;database`
```
Comment (illfated) [inline contrib/Docker/README.md:17]: Upon reflection, should the command line be encapsulated in double quotes to match the .conf file content format?

**Comment by jackpoz** [comment]
> * Replace HTML entity `&#46;` with its ASCII representation, a period dot.

On VS Code this makes README.md a clickable link which I wanted to avoid:
image
but it seems github doesn't mark it as link

File: contrib/Docker/README.md [hunk 2]
```diff
@@ -1,54 +1,62 @@
...
-
-The instructions below expect basic knowledge about how to configure TrinityCore and how to use Docker:
-1. Click on the green ✔ next to each commit
-1. Scroll to "ci/circleci: pch" and click on "Details"
-1. Login to Circle CI if necessary. You might have to repeat again the previous steps after logging in to reach the correct page
-1. Click on "Artifacts" tab in Circle CI website
-1. Download the docker.tar.gz archive containing the docker image
-1. Load the image into your Docker host using
+
+The Circle CI Linux PCH job uses the Dockerfile contained in the same folder as this README to create an image with the binaries built for Linux, and stores that in the job artifacts.
```
Comment (jackpoz) [inline contrib/Docker/README.md:3]: the job is actually called "pch" lowercase
image

File: contrib/Docker/README.md (hunk 1 above)
Comment (jackpoz) [reply contrib/Docker/README.md:17]: whatever looks ok on github is fine

**Comment by illfated** [comment]
> On VS Code this makes README.md a clickable link which I wanted to avoid:

OK, I see what you mean. Using the HTML entity is not an important issue. We can revert that change if you want to, in case it may be a source of usage issues.

File: contrib/Docker/README.md (hunk 2 above)
Comment (illfated) [reply contrib/Docker/README.md:3]: Fair point, I will revert that part.

File: contrib/Docker/README.md (hunk 1 above)
Comment (illfated) [reply contrib/Docker/README.md:17]: OK, well, I just wondered if that string is supposed to match the exact spelling/format in the .conf file. If you don't think it needs to be a carbon copy of that content, then I suppose it is fine as is.

Comment (jackpoz) [reply contrib/Docker/README.md:17]: could you post here an example as comment of what you mean, maybe with the 2 examples so I can see the difference ?

**Comment by jackpoz** [comment]
it's ok with the . in README.md

**Comment by illfated** [comment]
Hohoho... funny how I did not have the patience to wait for your comment... 😄

File: contrib/Docker/README.md (hunk 1 above)
Comment (illfated) [reply contrib/Docker/README.md:17]: https://github.com/TrinityCore/TrinityCore/blob/9af6bf15aa2fe836c3ebba306eaaa8971f00fac4/src/server/worldserver/worldserver.conf.dist#L95-L111

Comment (illfated) [reply contrib/Docker/README.md:17]:
```suggestion
1. Set the MySQL host in the .conf files to use the UNIX socket of MySQL, i.e.: `".;/var/run/mysqld/mysqld.sock;username;password;database"`
```

**Comment by jackpoz** [comment]
it sounds good to include " in the worldserver.conf

**Comment by illfated** [comment]
> it sounds good to include " in the worldserver.conf

🙂 Somewhat funny to see that I, being the PR author also added the comment suggestion and you, being the PR reviewer committed the suggestion, which is usually the opposite way compared to how it happens in most of the Microsoft Docs PR comments. 😄

**Comment by jackpoz** [comment]
Do the " show now or is there any other change left to add ?

**Review by ghost — COMMENTED** [review]
- **Hide whitespace changes:** https://github.com/TrinityCore/TrinityCore/pull/25864/files?diff=unified&w=1
- **Rich Diff view:** https://github.com/TrinityCore/TrinityCore/pull/25864/files?diff=unified&w=1&short_path=f042faa#diff-f042faa7196384b6cf55dca2276cf63d8bc2f7fa833733bd99781d3cda85298f

Yes, the quotes " " are now visible in the line

8. Set the MySQL host in the .conf files to use the UNIX socket of MySQL, i.e.: `".;/var/run/mysqld/mysqld.sock;username;password;database"`

edit: I have also edited the main PR comment to reference the correct change.

**Comment by illfated** [comment]
#sign-off

**Comment by jackpoz** [comment]
Thanks for the PR :)

**Pull request merged**